### ISSUE TITLE
dead is dead: allow for disabling mode on load

### ISSUE THREAD

**Issue by lynxlynxlynx** [ja2-stracciatella/ja2-stracciatella#824]
Testing bugs via provided save games from DoD mode is annoying, since they get overwritten all the time. It'd be ideal if they could be optionally treated as normal. Maybe a checkbox, maybe a compile time switch.

**Comment by RinneT** [comment]
I agree. I think I'd prefer a sort of 'unoffical' init parameter, as a specially compiled version seems a bit extreme, and a freely available option sort of defeats the purpose of the mode.

What would also be intriguing to me as a compile time switch would be the possibility of general debugging tools.
Meaning if someone ever felt like adding additional tools to debug a certain case, they could be made available with the switch, too.

**Comment by lynxlynxlynx** [comment]
We already have -debug for that, so you could just reuse it instead of an extra toggle.
